src/driver/mysql/MysqlDriver.ts

```diff
@@ -802,7 +808,7 @@ export class MysqlDriver implements Driver {
         } catch (e) {
             try {
                 this.mysql = PlatformTools.load("mysql2"); // try to load second supported package
-
+                this.version = 2;
```

Comment:
can we have the same order as above (first set version, then load packages) or vice-versa.. but not mixed?